elide-annotations/src/main/java/com/yahoo/elide/security/ChangeSpec.java

```diff
@@ -21,9 +21,17 @@
     @Override
     public String toString() {
         return String.format("ChangeSpec { resource=%s, field=%s, original=%s, modified=%s}",
-                resource,
+                safe(resource),
                 fieldName,
-                original,
-                modified);
+                safe(original),
+                safe(modified));
+    }
+
+    private String safe(Object object) {
```

Comment:
Is this really "safe"? Or are we doing something wrong and now swallowing that exception? i.e. if `object` is null.

Reply:
Null should be fine now.